### DISCUSSION TITLE
One of the key techniques we have used is NTK-aware interpolation (bloc97, 2023), which adjusts the scale to prevent the loss of high-frequency information in a training-free manner.

### DISCUSSION THREAD

**Discussion by eyuansu62** [QwenLM/Qwen#340] [General]
why the high-frequency information would be lost? 
Maybe the low-freq information?

**Comment by jklj077** [comment]
I think this was answered in https://github.com/QwenLM/Qwen/issues/343. I'll close the discussion for now.
